doc/compiler/tril/TrilLanguageReference.md

```diff
@@ -0,0 +1,228 @@
+<!--
+Copyright (c) 2016, 2017 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath 
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+-->
+
+# Tril Language Reference
+
+In may ways, the Tril language mirrors the Testarossa Intermediate Language,
+_Trees_. 
```

Comment:
Is _Trees_ the official name of the Testarossa IL?  I didn't think it had one other than "Testarossa IL".  People refer to "the trees" when talking about the IL, but I don't think that made its way into an official name.  Also, you don't refer to Testarossa IL as "Trees" subsequently in this document.

I'd say purge the ", _Trees_." from this sentence.

cc'ing Testarossa greybeards for opinion:   @mstoodle @vijaysun-omr @andrewcraik

Reply:
I have heard Trees as a proper name used for the IL with IBM in the past and I have seen documents that have used that name. That being said it isn't particularly distinctive. I'm on the fence about using here, but having a proper name for the IL would seem to be a good idea since what was Testarossa is now split over the OMR and OpenJ9 projects and having a name in the context of OMR that is inherited by others building on this technology would seem like a good idea to make documentation and discussion more clear going forward.

Reply:
There's a compromise point here: Could rephrase as 

> In may ways, the Tril language mirrors the Testarossa Intermediate Language,
colloquially known as _Trees_.